iothub/service/src/Jobs/ScheduledJobsOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class ScheduledJobsOptions
+    {
+        /// <summary>
+        /// Unique Job Id for the twin update job.
+        /// </summary>
+        [JsonProperty(PropertyName = "JobId", Required = Required.Default)]
+        public string JobId { get; set; }
+
+        /// <summary>
+        /// Max execution time in seconds, i.e., ttl duration the job can run.
+        /// </summary>
+        [JsonProperty(PropertyName = "MaxExecutionTimeInSeconds", Required = Required.Default)]
+        public long MaxExecutionTimeInSeconds { get; set; }
```

Comment:
I see this same property in the non-optional parameter classes like ScheduledDeviceMethod

Reply:
It can either be optional or required, but not both